superset-frontend/src/components/ProgressBar/index.tsx

```diff
@@ -46,8 +45,9 @@ const ProgressBar = styled(({ striped, ...props }: ProgressBarProps) => (
             transparent 25%, transparent 50%,
             rgba(255, 255, 255, 0.15) 50%,
             rgba(255, 255, 255, 0.15) 75%,
-            transparent 75%, transparent);
-        background-size: 1rem 1rem; `};
+            transparent 75%, transparent) !important;
```

Comment:
@rusackas asked me on a previous PR to avoid `!important`. Can you achieve the same result using other ways? Maybe with class specificity, &&& trick, etc..

Reply:
Thanks for catching it @michael-s-molina and thanks for trying, @geido - we can't win 'em all, but stay vigilant! :)